packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -472,9 +472,7 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions) {
               const { attendees: _attendees, ...bookingWithoutAttendees } = bookings;
               return bookingWithoutAttendees;
             }),
-          busyTimesFromLimitsBookings: busyTimesFromLimitsBookingsAllUsers.filter(
-            (b) => b.userId === currentUser.id
-          ),
+          busyTimesFromLimitsBookings: busyTimesFromLimitsBookingsAllUsers,
```

Comment:
The booking limit is event type wide and it doesn't matter which user was booked.